board/safety/safety_honda.h

```diff
@@ -22,13 +22,13 @@ static void honda_rx_hook(CAN_FIFOMailBox_TypeDef *to_push) {
   // sample speed
   if (addr == 0x158) {
     // first 2 bytes
-    honda_ego_speed = to_push->RDLR & 0xFFFF;
+    honda_ego_speed = GET_BYTE(to_push, 0) | GET_BYTE(to_push, 1);
```

Comment:
maybe `honda_moving` is a more appropriate name?

Reply:
done